packages/web-components/src/components/va-combo-box/va-combo-box.scss

```diff
@@ -0,0 +1,59 @@
+@forward 'settings';
+
+@use 'usa-label/src/styles/usa-label';
+@use 'usa-error-message/src/styles/usa-error-message';
+@use 'usa-combo-box/src/styles';
+@use 'uswds-helpers/src/styles/usa-sr-only';
+@use 'usa-icon/src/styles/usa-icon';
+@use 'usa-select/src/styles/usa-select';
+@use 'usa-input/src/styles/usa-input';
+@use 'usa-hint/src/styles/usa-hint';
+
+@import '../../mixins/uswds-input-width.scss';
+@import '../../mixins/uswds-error-border.scss';
+
+
+:host {
+  display: block;
+  font-family: var(--font-source-sans);
+  color: var(--vads-color-base);
+  width: 30rem;
+}
+
+:host([inert]) select {
+  border: 0;
+  background: none;
+}
+
+.usa-select {
+  margin-bottom: 0;
+}
+
+::slotted(option),
+::slotted(optgroup){
+  display: none;
+}
+
+.usa-combo-box{
+  font-size: 12px;
+  max-width: 100%;
+  width: 100%;
+  box-sizing: border-box;
+}
```

Comment:
I'm curious why we need to override the USWDS styling class here?

Reply:
I think I may have had some styling issues when testing with `yarn serve` instead of storybook. I will give it a test